eventhubs-streamanalytics-eventhubs/tools/eh-asa-perfmon/EventHandler.cs

```diff
@@ -0,0 +1,97 @@
+﻿using Azure.Messaging.EventHubs.Consumer;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace StreamingAtScale
+{
+    public class EventReceiver
```

Comment:
nit:  The name of the class should match the name of the file.

Reply:
resolved